src/services/user.ts

```diff
@@ -262,13 +262,16 @@ const getProfileInfo = async (username: string = ''): Promise<Profile | undefine
   if (!profile) return undefined;
 
   const fullName = `${profile?.FirstName} ${profile?.LastName}`;
+  const getCliftonStrengths = await CliftonStrengthsService.getCliftonStrengths(
```

Comment:
I wouldn't call this `getCliftonStrengths`. That name implies that it is a function, not a variable.
```suggestion
  const cliftonStrengths = await CliftonStrengthsService.getCliftonStrengths(
```

Reply:
That makes sense. Thank you!